architecture/additional_concepts/ephemeral-storage.adoc

```diff
@@ -82,3 +82,26 @@ exists, the `root` partition does not hold any image layer or other writable sto
 ====
 When you use DeviceMapper to provide runtime storage, a containers' copy-on-write layer is not accounted for in ephemeral storage management. Use overlay storage to monitor this ephemeral storage.
 ====
+
+[[section-managing-ephemeral-storage]]
+== Managing ephemeral storage
+
+Cluster administrators can manage ephemeral storage within a project by xref:../../admin_guide/quota.html#overview[setting quotas] that define the limit ranges and number of requests for ephemeral storage across all Pods in a non-terminal state. Developers can also xref:../../dev_guide/compute_resources.html#dev-compute-resources[set requests and limits] on this compute resource at the Pod and Container level.
+
+[[section-monitoring-ephemeral-storage]]
+== Monitoring ephemeral storage
+
+You can use `/bin/df` as a tool to monitor ephemeral storage usage on the volume where ephemeral container data is located, which is `/var/lib/docker` and `/var/lib/kubelet`. The available space for only `/var/lib/kubelet` is shown when you use the `df` command if `/var/lib/docker` is placed on a separate disk by the cluster administrator.
+
```

Comment:
I checked the ephemeral storage on a 3.11 cluster, looks the dirs for ephemeral storage  are `/var/lib/docker` and `/var/lib/origin`.

Reply:
Thanks for checking this @qinpingli. I'll update the text from `/var/lib/kubelet` to `/var/lib/origin/`.